lmp/tokenizer/_base_list_tokenizer.py

```diff
@@ -356,7 +405,19 @@ def build_vocab(
             min_count:
                 Minimum of token's frequence. If token's frequence is smaller
                 than `min_count`, then discard that token.
+
+        Raises:
+            TypeError:
+                When `batch_sequences` is not instance of `List[str]`.
+                When `min_count` is not instance of `int`.
```

Comment:
When `batch_sequences` is not instance of `List[str]` or `min_count` is not instance of `int`.